### PR TITLE
New package: mako-1.1

### PR DESCRIPTION
**Pull request by skrps**

### CODE REVIEW

**Comment by maxice8** [comment]
At the current state it will link against the whole elogind stack and require a rebuild when libelogind is made a separate package.

**Comment by skrps** [comment]
This package can wait until there.

**Comment by skrps** [comment]
mako builds here with libelogind. I can't test it locally because I'm not using Sway 1.0-alpha now; maybe I can test later.

**Comment by maxice8** [comment]
add wayland-protocols to makedepends ?

File: srcpkgs/mako/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mako'
+pkgname=mako
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc wayland-protocols"
+makedepends="wayland-devel pango-devel cairo-devel elogind-devel"
```
Comment (maxice8) [inline srcpkgs/mako/template]: add wayland-protocols to makedepends=

File: srcpkgs/mako/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mako'
+pkgname=mako
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc wayland-protocols"
```
Comment (maxice8) [inline srcpkgs/mako/template]: add wayland-devel to hostmakedepends=

File: srcpkgs/mako/template (hunk 1 above)
Comment (skrps) [reply srcpkgs/mako/template]: I was going to ask just that...

File: srcpkgs/mako/template (hunk 2 above)
Comment (skrps) [reply srcpkgs/mako/template]: Now I'm confused: should I add or exchange wayland-protocols and wayland-devel instead? Sorry, but communication sucks

Comment (maxice8) [reply srcpkgs/mako/template]: i didn't say anything about exchanging and removing both hostmakedepends and makedepends should have wayland-devel and wayland-protocols

Comment (skrps) [reply srcpkgs/mako/template]: ok, now it should be fixed

File: srcpkgs/mako/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'mako'
+pkgname=mako
+version=1.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc wayland-devel wayland-protocols"
+makedepends="wayland-devel wayland-protocols pango-devel cairo-devel
+ elogind-devel"
+short_desc="A lightweight Wayland notification daemon"
```
Comment (maxice8) [inline srcpkgs/mako/template]: please don't use A <thing> just use <thing>

"Lightweight Wayland notification daemon"

works perfectly fine

**Comment by skrps** [comment]
1 month bump :birthday:

**Comment by maxice8** [comment]
Did you test with sway alpha ?

**Comment by skrps** [comment]
Yep, running smoothly.

**Pull request merged**